warehouse/accounts/views.py

```diff
@@ -116,7 +116,7 @@ def login(request, redirect_field_name=REDIRECT_FIELD_NAME,
         # the index instead.
         if (not redirect_to or
                 not is_safe_url(url=redirect_to, host=request.host)):
-            redirect_to = "/"
+            redirect_to = "/manage/projects"
```

Comment:
I think it would be better if we used `request.route_path()` to generate a path to the `manage.projects` route instead of just setting this to be the string `"/manage/projects"`.

The behavior wouldn't change, but if this route ever changed in the future, we wouldn't have to come back here and update this line.

Reply:
Thanks for the review @di 

I'll address the reviews.